build/nodes-and-endpoints/join-mainnet.md

```diff
@@ -275,54 +275,54 @@ For more information regarding **Cosmovisor** please visit https://docs.cosmos.n
-	```bash
-	mkdir -p ~/.persistenceCore
-	mkdir -p ~/.persistenceCore/cosmovisor
-	mkdir -p ~/.persistenceCore/cosmovisor/genesis
-	mkdir -p ~/.persistenceCore/cosmovisor/genesis/bin
-	mkdir -p ~/.persistenceCore/cosmovisor/upgrades
-	```
+   ```bash
+   mkdir -p ~/.persistenceCore
+   mkdir -p ~/.persistenceCore/cosmovisor
+   mkdir -p ~/.persistenceCore/cosmovisor/genesis
+   mkdir -p ~/.persistenceCore/cosmovisor/genesis/bin
+   mkdir -p ~/.persistenceCore/cosmovisor/upgrades
+   ```
 
 3. Copy the current persistenceCore binary into the cosmovisor/genesis folder and the folder (**ensure you replace `BINARY_VERSION` with the appropriate upgrade version**)
 
-	```bash
-	export BINARY_VERSION=vX.Y.Z # IMPORTANT: REPLACE THIS VERSION WITH CORRECT UPGRADE VERSION
-	cp $GOPATH/bin/persistenceCore ~/.persistenceCore/cosmovisor/genesis/bin
-	mkdir -p ~/.persistenceCore/cosmovisor/upgrades/$BINARY_VERSION/bin
-	cp $GOPATH/bin/persistenceCore ~/.persistenceCore/cosmovisor/upgrades/$BINARY_VERSION/bin
-	```
+   ```bash
+   export BINARY_VERSION=vX.Y.Z # IMPORTANT: REPLACE THIS VERSION WITH CORRECT UPGRADE VERSION
+   cp $GOPATH/bin/persistenceCore ~/.persistenceCore/cosmovisor/genesis/bin
+   mkdir -p ~/.persistenceCore/cosmovisor/upgrades/$BINARY_VERSION/bin
+   cp $GOPATH/bin/persistenceCore ~/.persistenceCore/cosmovisor/upgrades/$BINARY_VERSION/bin
+   ```
 
 4. Set these environment variables. Copy and paste the following commands into your terminal.
 
-	```bash
-	echo "# Setup Cosmovisor" >> ~/.profile
-	echo "export DAEMON_NAME=persistenceCore" >> ~/.profile
-	echo "export DAEMON_HOME=$HOME/.persistenceCore" >> ~/.profile
-	echo "export DAEMON_ALLOW_DOWNLOAD_BINARIES=false" >> ~/.profile
-	echo "export DAEMON_LOG_BUFFER_SIZE=512" >> ~/.profile
-	echo "export DAEMON_RESTART_AFTER_UPGRADE=true" >> ~/.profile
-	echo "export UNSAFE_SKIP_BACKUP=true" >> ~/.profile
-	source ~/.profile
-	```
+   ```bash
+   echo "# Setup Cosmovisor" >> ~/.profile
+   echo "export DAEMON_NAME=persistenceCore" >> ~/.profile
+   echo "export DAEMON_HOME=$HOME/.persistenceCore" >> ~/.profile
+   echo "export DAEMON_ALLOW_DOWNLOAD_BINARIES=false" >> ~/.profile
+   echo "export DAEMON_LOG_BUFFER_SIZE=512" >> ~/.profile
+   echo "export DAEMON_RESTART_AFTER_UPGRADE=true" >> ~/.profile
+   echo "export UNSAFE_SKIP_BACKUP=true" >> ~/.profile
+   source ~/.profile
+   ```
 
 5. Now you can start **Cosmovisor**.
-	```bash
-	cosmovisor run start
-	```
+   ```bash
+   cosmovisor run start
+   ```
 
-Now, with **Cosmovisor** running our Persistence Core-1 Node, everything is set up correctly for when the next upgrade becomes available. Once an upgrade is available, please proceed to the following section **(Upgrade the Node)**.
+Now, with **Cosmovisor** running our Persistence test-core-1 Node, everything is set up correctly for when the next upgrade becomes available. Once an upgrade is available, please proceed to the following section **(Upgrade the Node)**.
```

Comment:
This needs to remain core-1 node since it's about mainnet.

Reply:
noted